pkgs/applications/virtualization/docker/default.nix

```diff
@@ -272,15 +272,15 @@ rec {
   # Get revisions from
   # https://github.com/moby/moby/tree/${version}/hack/dockerfile/install/*
   docker_20_10 = callPackage dockerGen rec {
-    version = "20.10.23";
+    version = "20.10.24";
     cliRev = "v${version}";
-    cliHash = "sha256-fNaRpstyG90Jzq3+U2A42Jj+ixb+m7tXLioIcsegPbQ=";
+    cliHash = "sha256-Q3J5xJ0k70s56dl+Nxp9mc7ig6F9A2cbRNvwfLvpjzo=";
     mobyRev = "v${version}";
-    mobyHash = "sha256-nBPw/M4VC9XeZ9S33HWdWSjY2J2mYpI/TPOzvLjSmJM=";
+    mobyHash = "sha256-7s9RfLsw88aCsHr1ldjTU2Dx66Edv51Flh4uIsmsBYY=";
     runcRev = "v1.1.4";
     runcHash = "sha256-ougJHW1Z+qZ324P8WpZqawY1QofKnn8WezP7orzRTdA=";
-    containerdRev = "v1.6.15";
-    containerdHash = "sha256-Vlftq//mLYZPoT2R/lHJA6wLnqiuC+Cpy4lGQC8jCPA=";
+    containerdRev = "v1.6.16";
```

Comment:
And this to 1.6.20

Reply:
I see 1.6.16 on the 20.10.14 tag https://github.com/moby/moby/blob/v20.10.24/hack/dockerfile/install/containerd.installer

Did I miss something?